core/controllers/admin.py

```diff
@@ -62,6 +66,12 @@
 
 from typing import Dict, List, Optional, TypedDict, Union, cast
 
+# Platform paramters that we plan to show on the the release-coordinator page.
+PLATFORM_PARAMS_TO_SHOW_IN_RC_PAGE = [
+    platform_parameter_list.ParamNames.PROMO_BAR_ENABLED.value,
+    platform_parameter_list.ParamNames.PROMO_BAR_MESSAGE.value
+]
```

Comment:
```suggestion
PLATFORM_PARAMS_TO_SHOW_IN_RC_PAGE = {
    platform_parameter_list.ParamNames.PROMO_BAR_ENABLED.value,
    platform_parameter_list.ParamNames.PROMO_BAR_MESSAGE.value
}
```

Should we make this a set? So the condition of checking `param['name'] not in PLATFORM_PARAMS_TO_SHOW_IN_RC_PAGE` is better with a set?

Reply:
Done